public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -151,11 +152,12 @@ export function DashboardSettings({ dashboard, editview }: Props) {
   const canSaveAs = contextSrv.hasEditPermissionInFolders;
   const canSave = dashboard.meta.canSave;
   const styles = getStyles(config.theme2);
+  const dashboardTitle = useSelector((state: StoreState) => state.dashboard.title);
```

Comment:
was thinking we would make state.dashboard keyed by uid (or something) before we start this journey so we can support more than one dashboard in state at the same time